src/Application/Models/Asset.js

```diff
@@ -1,13 +1,15 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked) {
+  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
     this._name = name;
     this._price = price;
     this._multiplier = multiplier;
     this._owned = owned;
     this._unlocked = unlocked;
     this._defaults = assetDefaults[name];
+    this._investment = investment;
```

Comment:
also, another nit: `unlocked` is a boolean for whether or not the player can purchase this Asset, correct?  Is `owned` a boolean for whether the player owns this Asset?  I think from outside knowledge that it's probably the number of this type of Asset that is owned.  If that's true, then the naming is confusing, since you're using the same naming pattern for a number and a boolean.  `owned` would much more naturally be `numOwned` then.